services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/InternalRepositoryUtils.java

```diff
@@ -142,8 +143,9 @@ static TableDto convertToTableDto(
             .tableUUID(megaProps.get(getCanonicalFieldName("tableUUID")))
             .tableLocation(
                 URI.create(
```

Comment:
Should probably encapsulate this logic in the same place as the other concatenation logic (i.e. TableLocation class).

Reply:
Because this return value is coming from Iceberg, it's in string format. Even if we add a TableLocation class, we'll then need to define `TableLocation.parse(String)`. String seems simpler and easier to reason with **For now**. Can re-consider if we see tablelocation usage proliferate.

Reply:
Discussed offline, see summary: https://github.com/linkedin/openhouse/pull/119#discussion_r1631640467